packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -391,11 +391,12 @@ mixin SchedulerBinding on BindingBase {
   AppLifecycleState? get lifecycleState => _lifecycleState;
   AppLifecycleState? _lifecycleState;
 
-  /// Allows the test framework to reset the lifecycle state back to its
-  /// initial value.
+  /// Allows the test framework to reset the lifecycle state and framesEnabled
+  /// back to their initial values.
   @visibleForTesting
-  void resetLifecycleState() {
+  void clearState() {
```

Comment:
Maybe `resetInternalState`? Some future state might not be "cleared" when it is reset, it might be returned to an initial value instead.

Reply:
Done!